tests/components/wallbox/test_lock.py

```diff
@@ -0,0 +1,108 @@
+"""Test Wallbox Lock component."""
+import json
+
+import pytest
+import requests_mock
+
+from homeassistant.components.lock import SERVICE_LOCK, SERVICE_UNLOCK
+from homeassistant.components.wallbox import CONF_LOCKED_UNLOCKED_KEY
+from homeassistant.const import ATTR_ENTITY_ID
+
+from tests.components.wallbox import entry, setup_integration
+from tests.components.wallbox.const import (
+    CONF_ERROR,
+    CONF_JWT,
+    CONF_MOCK_LOCK_ENTITY_ID,
+    CONF_STATUS,
+    CONF_TTL,
+    CONF_USER_ID,
+)
+
+authorisation_response = json.loads(
+    json.dumps(
+        {
+            CONF_JWT: "fakekeyhere",
+            CONF_USER_ID: 12345,
+            CONF_TTL: 145656758,
+            CONF_ERROR: "false",
+            CONF_STATUS: 200,
+        }
+    )
+)
+
+
+async def test_wallbox_lock_class(hass):
```

Comment:
You can add annotations to tests, too. Although they aren't checked with mypy, they can still help with autocompletions.

Reply:
Would you mind adding the annotations for `hass: HomeAssistant` in this file?

Reply:
OK, shall I create a seperate PR to also do it in the other test files?

Reply:
> OK, shall I create a seperate PR to also do it in the other test files?

A separate PR for the other ones is probably better. This one is close to being finished, let's do that afterwards.